Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml

```diff
@@ -7,7 +7,7 @@
         - to: girder
           completed:
             - !type:SnapToGrid
-              southRotation: true
+              southRotation: false
```

Comment:
Girders should still be south-facing.

Reply:
This one is necessary for constructing the diagonal walls. If it snaps to South, the walls will only construct in South position.

Reply:
Ok, with further testing, I might have been wrong. Going to double test. But yeah, seems this is unnecessary.